subsystem/src/main/java/org/wildfly/extension/grpc/GrpcServerConfigService.java

```diff
@@ -0,0 +1,69 @@
+/*
+ *  Copyright 2022 Red Hat
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wildfly.extension.grpc;
+
+import org.jboss.msc.service.Service;
+import org.jboss.msc.service.ServiceBuilder;
+import org.jboss.msc.service.ServiceController;
+import org.jboss.msc.service.ServiceName;
+import org.jboss.msc.service.ServiceTarget;
+import org.jboss.msc.service.StartContext;
+import org.jboss.msc.service.StartException;
+import org.jboss.msc.service.StopContext;
+
+/**
+ * WS server config service.
+ *
+ * @author <a href="alessio.soldano@jboss.com">Alessio Soldano</a>
+ * @author <a href="mailto:ropalka@redhat.com">Richard Opalka</a>
+ * @author <a href="mailto:ema@redhat.com">Jim Ma</a>
+ * @author <a href="mailto:rsigal@redhat.com">Ron Sigal</a>
+ */
+@SuppressWarnings("deprecation")
+public final class GrpcServerConfigService implements Service<GrpcServerConfig> {
```

Comment:
I don't see why we'd need a service here. The start/stop don't really do anything.

Reply:
Mmmm, no idea ...

Reply:
Here we're just storing a configuration which is something I don't really think we need. We can interact directly with the `GrpcServerService` which, IMO, would be the preferred way.